code/renderers/svelte/src/docs/extractArgTypes.ts

```diff
@@ -37,7 +37,7 @@ export const createArgTypes = (docgen: SvelteComponentDoc) => {
   if (docgen.data) {
     docgen.data.forEach((item) => {
       results[item.name] = {
-        control: parseTypeToControl(item.type),
+        ...parseTypeToControl(item.type),
```

Comment:
This change and the corresponding change within `parseTypeToControl` wasn't necessary, was it? To my naked eye it seems we're doing the exact same thing, still.

Reply:
Actually, this change is necessary because we need to place `options` at a higher level. The current `options` within `control` do not seem to be working.

```
      return {
        control: { type: 'radio' },
        options: type.type.map((t) => (t as JSDocTypeConst).value),
      };
```

Reply:
I see. Weird, but unrelated to this PR, so I'm fine with this.